tst/testinstall/mapping.tst

```diff
@@ -287,6 +287,17 @@ gap> IsGroupHomomorphism(res);
 true
 gap> IsInjective(res);        
 true
+
+# The following test verifies we handle pcgs with large primes in them efficiently.
+# See also GitHub pull request #576
```

Comment:
Or maybe mention both #576 and #578, since the discussion happened in the other one.